crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/AssessmentCompletionServiceTest.java

```diff
@@ -0,0 +1,160 @@
+package uk.gov.justice.laa.crime.meansassessment.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.justice.laa.crime.meansassessment.data.builder.TestModelDataBuilder;
+import uk.gov.justice.laa.crime.meansassessment.dto.MeansAssessmentDTO;
+import uk.gov.justice.laa.crime.meansassessment.dto.maatcourtdata.DateCompletionRequestDTO;
+import uk.gov.justice.laa.crime.meansassessment.dto.maatcourtdata.FinancialAssessmentDTO;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AssessmentCompletionServiceTest {
+
+    private MeansAssessmentDTO assessment;
+    private final String LAA_TRANSACTION_ID = "laa-transaction-id";
+
+    @Spy
+    @InjectMocks
+    private AssessmentCompletionService assessmentCompletionService;
+
+    @Mock
+    private MaatCourtDataService maatCourtDataService;
+
+
+    @Before
+    public void setup() {
+        assessment = TestModelDataBuilder.getMeansAssessmentDTO();
+    }
+
+    @Test
+    public void givenNullExistingCompletionDate_whenIsFullUpdateRequiredIsInvoked_thenReturnTrue() {
+        when(maatCourtDataService.getFinancialAssessment(anyInt(), anyString()))
+                .thenReturn(FinancialAssessmentDTO.builder().build());
+
+        boolean result = assessmentCompletionService.isFullUpdateRequired(assessment, LAA_TRANSACTION_ID);
+        assertThat(Boolean.TRUE).isEqualTo(result);
```

Comment:
Maybe we can keep this short like (just a suggestion):
assertThat(result).isTrue();

This applies to all the Boolean comparisons in this Test class.

Reply:
Good suggestion, done :)